defi/src/protocols/data3.ts

```diff
@@ -23799,20 +23799,22 @@ const data3: Protocol[] = [
     address: null,
     symbol: "-",
     url: "https://marginly.com",
-    description: "Margin trading and derivatives on top of AMM liquidity.",
+    description: "Get up to 50x leverage on trading, farming, staking, on anything.",
     chain: "Arbitrum",
     logo: `${baseIconsUrl}/marginly.jpg`,
     audits: "2",
     audit_note: null,
     gecko_id: null,
     cmcId: null,
-    category: "Derivatives",
-    chains: ["Arbitrum"],
+    category: "Leveraged Farming",
```

Comment:
remove the category change
what we track on the current adapter is derivatives

Reply:
How it could be changed to "Leveraged Farming" ?